### ISSUE TITLE
Fails to log in with an incorrect password

### ISSUE THREAD

**Issue by jpkim921** [learn-co-students/sinatra-user-auth-v-000#877]
the code along isn't correct.

it leaves out the if statement to check if the password is correct.

**Comment by indykiss** [comment]
Agreed! This seems to still be an issue, no codealong for the password being correct path.
